Lib/pstats.py

```diff
@@ -333,6 +333,45 @@ def eval_print_amount(self, sel, list, msg):
 
         return new_list, msg
 
+    def get_profile_dict(self, keys_filter=None):
```

Comment:
Document the new method in Doc/library/profile.rst, including a versionadded 3.9 tag.

Reply:
Done